src/cpp/r/RSexp.cpp

```diff
@@ -564,9 +587,20 @@ SEXP functionBody(SEXP functionSEXP)
    return bodySEXP;
 }
 
-SEXP findVar(const std::string &name, const SEXP env)
+SEXP findVar(SEXP nameSEXP, SEXP envSEXP)
 {
-   return Rf_findVar(Rf_install(name.c_str()), env);
+#ifndef RSTUDIO_PACKAGE_BUILD
```

Comment:
This is mainly for internal use; in general, we try to avoid introspecting an active binding as it will cause that binding to be "fired" which we typically want to avoid.